indigo_app/notifications.py

```diff
@@ -115,6 +140,12 @@ def notify_comment_posted(comment_id):
         log.warning("Comment with id {} doesn't exist, ignoring".format(comment_id))
 
 
+@receiver(post_save, sender=Annotation)
+def post_annotation_reply(sender, **kwargs):
```

Comment:
I've made some changes to the notification signals, so that they can be sent by a background thread. Otherwise, if the SMTP service (eg. mailgun) is having problems, it will cause the action (eg. creating an annotation) to fail.

1. please move the signal receiver into `signals.py`
2. add a new function here which is setup as a background task: it should take the annotation id, load the object from the DB, and then pass that to the notifier method.

See the `notify_comment_posted` method as an example.